docker/README.md

```diff
@@ -0,0 +1,19 @@
+## Docker based environment
+
+### Building a docker image
+
+From the root directory of this repository (wlan-testing) run the following command:
+```bash
+docker build -f ./docker/Dockerfile -t wlantest
```

Comment:
Think it should be `docker build -f ./docker/Dockerfile -t wlantest .`, right?

Reply:
Thats right. Will make that fix.